### PR TITLE
User-set Proposal Titles, pt 1: DB modifications

### PR DESCRIPTION
**Pull request by gdjohnson**
Per [spec](https://www.notion.so/hicommonwealth/Chain-Entity-Authors-Titles-13-2a0389729cb544c79ab44258b4fd30f8): Add title/author column to ChainEntities in db. To test: set a title manually on the Server.

Titles + authors have both been set manually from Postico without issue. 

## Have proper tags been added (for bug, enhancement, breaking change)?
- [x] yes

### CODE REVIEW

File: server/migrations/20210203200024-add-title-to-chain-entities.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.addColumn(
+          'ChainEntities',
+          'title',
+          {
+            type: Sequelize.TEXT,
```
Comment (jnaviask) [inline server/migrations/20210203200024-add-title-to-chain-entities.js:11]: Should be STRING (unless there's a good reason to use TEXT here?).

File: server/migrations/20210203200024-add-title-to-chain-entities.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    });
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.removeColumn(
+          'ChainEntities',
+          'title',
+          {
```
Comment (jnaviask) [inline server/migrations/20210203200024-add-title-to-chain-entities.js:35]: I think you need to remove the column specification for these removeColumn commands.

**Review by jnaviask — CHANGES_REQUESTED** [review]
Small fixes.

File: server/migrations/20210203200024-add-title-to-chain-entities.js (hunk 1 above)
Comment (gdjohnson) [reply server/migrations/20210203200024-add-title-to-chain-entities.js:11]: I was just mirroring OffchainThread model! I'm happy with string. STRING defaults to a max length of 255, whereas TEXT is basically VarChar?

File: server/migrations/20210203200024-add-title-to-chain-entities.js (hunk 2 above)
Comment (gdjohnson) [reply server/migrations/20210203200024-add-title-to-chain-entities.js]: Done!

File: server/migrations/20210203200024-add-title-to-chain-entities.js (hunk 1 above)
Comment (jnaviask) [reply server/migrations/20210203200024-add-title-to-chain-entities.js]: Yep -- that's the distinction. I assume we'll be fine with titles maxing out at 255, but ofc if we want to support very long titles... then we can make it TEXT.

**Review by raykyri — APPROVED** [review]

**Pull request closed without merging**